dev/packages/example/endpoint-1.0.0/kibana/index-pattern/events-endpoint-*.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
@jonathan-buttner I would assume this index pattern is not needed because it is generated by EPM? Or is there something more specific about this index pattern then `events-*`?

Reply:
Oh good point, I'll switch it to use `events-*`